exchanges/exchange_test.go

```diff
@@ -1098,21 +1098,19 @@ func TestUpdatePairs(t *testing.T) {
 		t.Fatalf("received: '%v' but expected: '%v'", false, true)
 	}
 
-	// purge all pairs
 	err = UAC.UpdatePairs(currency.Pairs{}, asset.Spot, true)
-	require.NoError(t, err)
+	require.NoError(t, err, "purging all pairs must not error")
 
-	unset, err := UAC.GetEnabledPairs(asset.Spot)
+	pairs, err = UAC.GetEnabledPairs(asset.Spot)
 	require.NoError(t, err)
-	require.Empty(t, unset)
+	require.Empty(t, pairs)
 
-	// preserve existing pairs for available list
 	err = UAC.UpdatePairs(currency.Pairs{}, asset.Spot, false)
-	require.ErrorIs(t, err, currency.ErrCurrencyPairsEmpty)
+	require.ErrorIs(t, err, currency.ErrCurrencyPairsEmpty, "Purging all available pairs must error")
 
 	avail, err := UAC.GetAvailablePairs(asset.Spot)
 	require.NoError(t, err)
-	require.NotEmpty(t, avail)
+	assert.NotEmpty(t, avail, "Failed attempt to purge available pairs should not effect store")
```

Comment:
Sorryyyyyyyy 🫤 

```suggestion
	assert.NotEmpty(t, avail, "Failed attempt to purge available pairs should not affect store")
```

Reply:
who really knows the diff if you talk real fast anyway 🤷